src/components/MetaHeadEmbed.tsx

```diff
@@ -158,10 +159,13 @@ const MetaHeadEmbed = ({
         content={commaSeparate(keywords)}
       />
     ),
-    canonicalUrl && <link key="canonical" rel="canonical" href={canonical} />,
+
+    !disableCanonical && (
+      <link key="canonical" rel="canonical" href={pathUrl} />
+    ),
 
     <meta key="og:type" property="og:type" content="website" />,
-    <meta key="og:url" property="og:url" content={pageUrl} />,
+    <meta key="og:url" property="og:url" content={pagePath} />,
```

Comment:
```suggestion
    <meta key="og:url" property="og:url" content={pageUrl} />,
```

Reply:
👍

Reply:
It's still `pagePath`? When it should be `pageUrl`?